src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,53 @@
+package org.example;
+
+import org.example.repository.GameRepositoryImpl;
+import org.example.repository.UserRepositoryImpl;
+import org.example.service.GameService;
+import org.example.service.UserService;
+import org.example.service.menu.MainServiceMenu;
+import org.example.service.menu.UserServiceManager;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        try (Connection connection = ConnectionSingleton.getConnection()) {
+            try (Scanner scanner = new Scanner(System.in)) {
+                scanner.useDelimiter("\n");
+                UserContext userContext = UserContext.getInstance();
+                UserService userService = new UserService(new UserRepositoryImpl(connection));
+                GameService gameService = new GameService(new GameRepositoryImpl(connection));
+
+                UserServiceManager management = new UserServiceManager(userService, scanner);
```

Comment:
Maybe it would be better to rename this variable.
```suggestion
                UserServiceManager userServiceManager = new UserServiceManager(userService, scanner);
```